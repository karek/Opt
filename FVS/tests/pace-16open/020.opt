8
15 61 50 14 19 51 30 57 
