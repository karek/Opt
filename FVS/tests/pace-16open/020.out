8
30 50 51 39 15 14 61 21 
